xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1210 @@
+-v `pwd`<replaceable>/config.airgapped.yml</replaceable>:/etc/docker/registry/config.yml \
+-v `pwd`/auth:/auth \
+-v `pwd`/certs:/certs \
+-v <replaceable>/registry</replaceable>:/var/lib/registry sles12/registry:2.6.2</command>
+       </screen>
+     </step>
+    </procedure>
+
+    <para>
+     Now, you should have the registries set up and listening on port
+     <literal>5000</literal> on their respective servers.
+    </para>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.client">
+    <title>Client Configuration</title>
+    <para>
+     Using the airgapped mirror works exactly like using a traditional
+     on-premise mirror. You must configure the internal registry mirror in
+     &dashboard;.
+    </para>
+    <procedure>
+     <title>Configuring Registry Mirror</title>
+     <step>
+      <para>
+       Log in to &dashboard; on the &productname; admin node.
+      </para>
+     </step>
+     <step>
+      <para>
+       Navigate to <guimenu>Settings &rarr; Mirrors</guimenu> and create a
+       definition for your internal registry mirror as described in
+       <xref linkend="sec.admin.velum.mirror"/>.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.helm-charts">
+   <title>Helm Chart Repository Mirror</title>
+   <important>
+    <para>
+     To make use of the helm charts, you must complete
+     <xref linkend="sec.deploy.scenarios.airgap.container-registry"/>.
+    </para>
+   </important>
+   <para>
+    The helm charts will require images available from a registry mirror. The
+    charts themselves are served on a simple webserver and do not require any
+    particular configuration apart from basic networking availability and a
+    hostname.
+   </para>
+   <sect3>
+    <title>Mirror Configuration</title>
+    <procedure>
+     <title>Setting Up Chart Repository Mirror</title>
+     <step>
+      <para>
+       Install and configure a webserver (for example <link xlink:href="https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_apache2.html">Apache</link>)
+       on your internal network. You can just deploy a webserver onto the
+       internal mirror.
+      </para>
+     </step>
+     <step>
+      <para>
+       Configure a hostname that will be used as the helm chart repository
+       address inside the internal network. This hostname must point to the
+       configured webserver. If you provide the webserver on the internal
+       mirror, we recommend configuring the webserver to be available at
+       <literal>/charts</literal> to avoid confusion with the image registry
+       endpoint. The Helm chart repository would then be accessible at
+       <literal>https://myregistry.local:443/charts/</literal>.
+       </para>
+       <para>
+        You will need the configured hostname to replace the original
+        chart repository address in the index file of your repository.
```

Comment:
We have done that earlier when using the `helm-mirror` command. I think it would be nice to tell the customer: look, you have already done that before, see `[here]` (with `here` being a link to the section of the document where we explained that).